compliance/utils/services/github.py

```diff
@@ -413,6 +414,56 @@ def get_commit_details(self, repo, since, branch='master', path=None):
             opts['path'] = path
         return self._make_request('get', f'repos/{repo}/commits', params=opts)
 
+    def get_pull_requests(self, repo, since=None, **kwargs):
+        """
+        Retrieve a repository's pull request information.
+
+        :param repo: the organization/repository as a string.
+        :param since: the starting date/time of a pull request as a datetime.
+
+        :returns: the repo branch's pull request information
+        """
```

Comment:
```suggestion
        """
        Retrieve a repository's pull request information.

        :param repo: the organization/repository as a string.
        :param since: the starting date/time of a pull request as a datetime.
        :param kwargs: key/value pairs of GH pulls API accepted params

        :returns: Repository pull request metadata
        """
```